packages/airnode-validator/README.md

```diff
@@ -7,8 +7,33 @@ them.
 ## Usage
 
 General usage of validator and convertor is described in the
-[API3 Documentation](https://docs.api3.org/airnode/latest/reference/packages/validator.html). For debugging purposes it
-might be useful to run the validator with path to template file instead of providing the format name:
+[API3 Documentation](https://docs.api3.org/airnode/latest/reference/packages/validator.html).
+
+## Build Manually
+
+You can clone and build the Airnode monorepo then run the validator as a yarn script from inside the
+`packages/airnode-validator` directory.
+
+```sh
+# download and build the airnode monorepo
+git clone git@github.com:api3dao/airnode.git
```

Comment:
I think this shouldn't be there, because adding it here implies adding it to every airnode package which is repetitive. These instructions are already part of https://github.com/api3dao/airnode#instructions

Reply:
I agree, opened a [new PR](https://github.com/api3dao/airnode/pull/755#issuecomment-1004054627) for this